benchmarks/kpi/README.md

```diff
@@ -0,0 +1,29 @@
+## KPI Anomaly dataset
```

Comment:
I think we will have to add this to copyright for our data: https://github.com/NetManAIOps/KPI-Anomaly-Detection/blob/master/LICENSE

Reply:
What do you mean by adding copyright? They have specified their license. And we are not adding their data in the repository. Just running the analysis on it.

Reply:
Added the license